src/services/swap_service.ts

```diff
@@ -367,6 +368,32 @@ export class SwapService implements ISwapService {
             sellTokenFeeAmount,
         } = serviceUtils.getAffiliateFeeAmounts(swapQuote, affiliateFee);
 
+        const affiliateFeeAmounts: AffiliateFeeAmount[] = [
+            {
+                recipient: affiliateFee.recipient,
+                feeType: affiliateFee.feeType,
+                buyTokenFeeAmount,
+                sellTokenFeeAmount,
+            },
```

Comment:
```suggestion
```
I'm not sure what this was for?

Reply:
I think this is how the old affiliate fee got initialized and passed in - don't want to change the behavior in case it bricks how affiliate fees work. Think we can probably refactor for clarity though

although, let me know if you see a logical error!